system/manager/process_config.py

```diff
@@ -63,14 +63,15 @@ def and_(*fns):
 procs = [
   DaemonProcess("manage_athenad", "system.athena.manage_athenad", "AthenadPid"),
 
-  NativeProcess("camerad", "system/camerad", ["./camerad"], driverview),
+  NativeProcess("camerad", "system/camerad", ["./camerad"], driverview, enabled=not WEBCAM),
+  PythonProcess("camerad", "tools.webcam.camerad", driverview, enabled=WEBCAM),
```

Comment:
```suggestion
  PythonProcess("webcamerad", "tools.webcam.camerad", driverview, enabled=WEBCAM),
```

Reply:
✅done

As far as I know, nothing relies on the hard-coded name `camerad`